adminSiteServer/apiRouter.ts

```diff
@@ -2450,16 +2451,16 @@ apiRouter.post("/posts/:postId/unlinkGdoc", async (req: Request) => {
 
 apiRouter.get("/sources/:sourceId.json", async (req: Request) => {
     const sourceId = expectInt(req.params.sourceId)
-    const source = await db.mysqlFirst(
+    const source = await db.knexRawFirst<Record<string, any>>(
```

Comment:
When it wasn't particularly important, I didn't use specific types. Let me know if you prefer to have strict types everywhere.